src/main/java/uk/gov/cslearning/record/api/InviteController.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.cslearning.record.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.cslearning.record.domain.Event;
+import uk.gov.cslearning.record.domain.Invite;
+import uk.gov.cslearning.record.repository.EventRepository;
+import uk.gov.cslearning.record.repository.InviteRepository;
+
+import java.util.Collection;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+@RestController
+@RequestMapping("/event")
+public class InviteController {
+
+    @Autowired
+    EventRepository eventRepository;
+
+    @Autowired
+    InviteRepository inviteRepository;
+
+    public InviteController(InviteRepository inviteRepository, EventRepository eventRepository){
+        checkArgument(inviteRepository != null);
+        checkArgument(eventRepository != null);
+        this.inviteRepository = inviteRepository;
+        this.eventRepository = eventRepository;
+    }
+
+    @GetMapping("/{eventId}/invitee")
+    public ResponseEntity<Collection<Invite>> listInvitees(@PathVariable("eventId") String catalogueId){
+        Collection<Invite> result = inviteRepository.findByEventId(catalogueId);
+        
+        return new ResponseEntity<>(result, HttpStatus.OK);
+    }
+
+    @PostMapping("/{eventId}/invitee")
+    public ResponseEntity<Event> addInvitee(@PathVariable("eventId") String catalogueId, @RequestBody Invite invite, UriComponentsBuilder builder){
```

Comment:
INFO - What is the CatelogueId rather than eventId?

Reply:
The catalogueId is the id of the event in learning-catalogue, as that's the id that identifies events in lpg-management, so will be the id that's passed to learner-record when find bookings and invites on an event

Reply:
but internally it is event Id right?